docs/recipes/atom-with-default-refresh.mdx

```diff
@@ -1,12 +1,10 @@
 ---
 title: atomWithRefreshAndDefault
```

Comment:
Is it intentional to change the file name from the title?
Can it be `atom-with-refresh-and-default.mdx`?

Reply:
Same for `atom-with-toggle-and-storage.mdx`

Reply:
That's a workaround for this issue I'm having when I name them like that

<img width="256" alt="image" src="https://github.com/pmndrs/jotai/assets/36211892/be953160-881c-4763-9b76-0af643074dfd">

Reply:
Not sure what you mean. My concern is the mdx file name.

Reply:
The issue I mentioned before is that on the sidebar it toggles two items selected when you go to atomWithToggleAndStorage when you name the file like `atom-with-toggle-and-storage.mdx` (in the screenshot you can see that atomWithToggle and atomWithToggleAndStorage are selected). Same happens with `atom-with-refresh-and-default.mdx`. 

I'll update it right away, but just so you know about it :)

Reply:
I see what you mean. It's the website issue. Let's merge this and fix it separately. @sandren can look into it?